src/app/account/code/code.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+
+import { AuthService } from 'app/shared/services/auth.service';
+import { LOGIN_ERROR_MESSAGES, RE_SEND_TIME } from '@constants/app-constants';
+import { finalize } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-code',
+  templateUrl: './code.component.html',
+  styleUrls: ['./code.component.scss']
+})
+export class CodeComponent implements OnInit {
+
+  public codeFormControl = new FormControl('', [
+    Validators.required,
+  ]);
+  public errorMessage: string;
+  public showError = false;
+  public timer: number;
+  public showTimer = true;
+  public reSendTimer: any;
+  public disableReSendBtn = false;
+
+  constructor(
+    private authService: AuthService,
+  ) { }
+
+  ngOnInit() {
+    this.startTimer();
+  }
+
+  startTimer() {
+    this.timer = this.authService.getEmailAndReSendTime().reSendTime;
+    this.showTimer = this.timer === 0 ? false : true;
+    this.reSendTimer = setInterval(() => {
+      this.timer--;
+      if (this.timer <= 0) {
+        this.stopTimer(this.reSendTimer);
+      }
+    }, 1000);
```

Comment:
Use from constants

Reply:
I don't think it needs because this represents 1 second.

Reply:
Please mention it here, that it represents 1 second.